scripts/check-g-cloud-live.py

```diff
@@ -2,7 +2,7 @@
 """
 
 Usage:
-    scripts/check-g-cloud-7-live.py <stage> <api_token> <draft_bucket> <documents_bucket>
```

Comment:
There are still a [couple](https://github.com/alphagov/digitalmarketplace-scripts/blob/1f99b4f67ad38fd8f0023ea8e181d29532652308/scripts/check-g-cloud-live.py#L30) of [places](https://github.com/alphagov/digitalmarketplace-scripts/blob/1f99b4f67ad38fd8f0023ea8e181d29532652308/scripts/check-g-cloud-live.py#L71) where `g-cloud-7` is hardcoded into it.

Is this file something we're still expecting to use?